Automattic/wp-calypso
# Gutenboarding - try changing fonts in Dynamic Preview

For summary of these `BlockPreview` draft experiments, please visit `pbAok1-lN-p2`.

#### Changes proposed in this Pull Request

Test altering global style fonts in dynamic preview as per #39513.

We will need to properly load in all style assets for the blocks, themes, and fonts for this to be fully functional, but currently we can make use of the global styles `--font-headings` and `--font-base` properties by toggling them between `serif` and `sans-serif`.

***Question*** - what are all the styles required to make this fully operational and how to we go about importing them all into Gutenboarding?  Currently we are importing the block-library styles as well as the exford theme css sheets (for the sake of having A theme css sheet), but the Exford preview is not fully styled and we must be missing some things...

#### Testing instructions

Currently missing some style assets which makes this a bit hard to see. but on the design selector with the preview you can:

* open the console
* reference the styles wrappers - `let node = document.getElementsByClassName( 'editor-styles-wrapper' );`
* change the style properties for base fonts and header fonts between serif and sans-serif 
* `node[1].style.setProperty( '--font-base', 'sans-serif' )` 
* `node[1].style.setProperty( '--font-base', 'serif' )` 
* `node[1].style.setProperty( '--font-headings', 'sans-serif' )` 
* `node[1].style.setProperty( '--font-headings', 'serif' )` 
* notice the changes take effect in *some* sections of the text in the preview



Fixes #
